packages/apps/frontera/tests/pages/allOrgsPage.ts

```diff
@@ -0,0 +1,166 @@
+import { Page, expect } from '@playwright/test';
+
+export class AllOrgsPage {
+  constructor(private page: Page) {}
+
+  async waitForPageLoad() {
+    const allOrgsMenubutton = this.page.locator('button:has-text("All orgs")');
+    await allOrgsMenubutton.waitFor({ state: 'visible' });
+    await expect(allOrgsMenubutton).toBeVisible();
+    await allOrgsMenubutton.click();
+  }
+
+  async addOrganization() {
+    const addOrganizationsButton = await this.page.waitForSelector(
+      'button.inline-flex.items-center.justify-center.font-semibold.text-gray-700.border.border-solid.border-gray-300.hover\\:bg-gray-50.hover\\:text-gray-700.focus\\:bg-gray-50.rounded-lg.text-sm',
+    );
+    // const addOrganizationsButton = await this.page.waitForSelector(
```

Comment:
is this supposed to be commented out?

Reply:
That is the work in progress. I wrote for myself how the new locator would look like. Will remove it to have the PR clean